gabbi/tests/test_utils.py

```diff
@@ -87,3 +93,42 @@ def test__colorize_missing_color(self):
 
         message = utils._colorize('BLUE', 'hello')
         self.assertNotEqual('hello', message)
+
+
+class CreateUrlTest(unittest.TestCase):
```

Comment:
I wish you were consistent in your use of capitals (where consistent here means `CreateURLTest`, because I said so).